src/modm/driver/display/ks0108.hpp

```diff
@@ -41,7 +40,7 @@ namespace modm
 	 */
 	template < typename E, typename RW, typename RS,
 			   typename PIN_CS1, typename PIN_CS2, typename PORT >
-	class Ks0108 : public BufferedGraphicDisplay<128, 64>
+	class Ks0108 : public MonochromeGraphicDisplayBufferedVertical<128, 64>
```

Comment:
```suggestion
	class Ks0108 : public MonochromeGraphicDisplayBufferedVerticalFactory<128, 64>
```
![2wbvgn](https://user-images.githubusercontent.com/163066/54542947-fd9a8980-499c-11e9-8a80-ac30616269c9.jpg)

Reply:
That are only 40 characters for that name, there is still room to grow :smile: